pkg/stdlib/html/html_parse.go

```diff
@@ -0,0 +1,30 @@
+package html
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/MontFerret/ferret/pkg/runtime/core"
+	"github.com/MontFerret/ferret/pkg/runtime/values"
+	"github.com/MontFerret/ferret/pkg/runtime/values/types"
+	"github.com/PuerkitoBio/goquery"
+	"github.com/pkg/errors"
+)
+
+func HtmlParse(ctx context.Context, args ...core.Value) (core.Value, error) {
+	err := core.ValidateType(args[0], types.String)
+
+	if err != nil {
+		return values.False, err
```

Comment:
Why are you returning `return values.False, err` here and `return nil, errors.Wrap(err, "failed to parse a document")` below? Doesn't look consistent.